### ISSUE TITLE
Unable to start asusctltray on endeavouros

### ISSUE THREAD

**Issue by ek234** [Baldomo/asusctltray#23]
Thanks for the application!
I am using endeavouros on asus g14. On running the asusctltray command in a terminal, I am greeted with this error:

```
ERROR:dbus.proxies:Introspect error on :1.4:/org/asuslinux/Profile: dbus.exceptions.DBusException: org.freedesktop.DBus.Error.UnknownObject: Unknown object '/org/asuslinux/Profile'
Traceback (most recent call last):
  File "/usr/local/bin/asusctltray", line 690, in <module>
    TrayIcon(loop)
  File "/usr/local/bin/asusctltray", line 650, in __init__
    self.build_menu()
  File "/usr/local/bin/asusctltray", line 666, in build_menu
    FanCurveController().attach(self.menu)
  File "/usr/local/bin/asusctltray", line 328, in attach
    for item in self.items:
                ^^^^^^^^^^
  File "/usr/local/bin/asusctltray", line 464, in items
    return [str(i) for i in self.proxy.Profiles()]
                            ^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/site-packages/dbus/proxies.py", line 72, in __call__
    return self._proxy_method(*args, **keywords)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/site-packages/dbus/proxies.py", line 141, in __call__
    return self._connection.call_blocking(self._named_service,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/site-packages/dbus/connection.py", line 634, in call_blocking
    reply_message = self.send_message_with_reply_and_block(
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
dbus.exceptions.DBusException: org.freedesktop.DBus.Error.UnknownObject: Unknown object '/org/asuslinux/Profile'
```
Lemme know if you want any other info...

**Comment by Baldomo** [comment]
Thank you for the report. It looks like asusctl's DBus API was changed. I don't have my G14 on hand so I cannot fix this at the moment, but I will investigate and update the script as soon as I can.

**Comment by Baldomo** [comment]
This has been fixed in c8ef0ba, tested on `asusctl` 6.0.7
